### PR TITLE
pulumi 2.9.1

### PR DESCRIPTION
**Pull request by stack72**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/pulumi.rb [hunk 1]
```diff
@@ -16,6 +16,26 @@ class Pulumi < Formula
...
+
+  resource "protobuf" do
+    url "https://files.pythonhosted.org/packages/25/ef/8f36bc7e4c3dcad241a9e7a51520d30712a7abc034343bed841ff4ea3289/protobuf-3.13.0.tar.gz"
+    sha256 "6a82e0c8bb2bf58f606040cc5814e07715b2094caeba281e2e7d0b0e2e397db5"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
+    sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
+  end
```
Comment (chenrui333) [inline Formula/pulumi.rb:37]: these are python dependencies, right?

I think you need specify `depends_on "python@3.8"`?

Comment (stack72) [reply Formula/pulumi.rb:37]: Hi @chenrui333 

So the bump-formula-pr tool added these by default - so I'd love some guidance here on what you need

Paul

Comment (chenrui333) [reply Formula/pulumi.rb:37]: ok, maybe we should remove these dependencies. cc @dtrodrigues

Comment (chenrui333) [inline Formula/pulumi.rb:37]:
```suggestion
```

**Comment by dtrodrigues** [comment]
Yeah these python dependencies shouldn't have been added by the command. `bump-formula-pr` searches PyPI for a package of the same name as the formula if the url isn't a `pythonhosted` URL. I submitted a PR to add pulumi to the blocklist -- we might not want to always fall back to the formula name to try and find it in PyPI.

**Comment by SMillerDev** [comment]
I think we should never do that, or only when we can reasonably identify it as a python package. Virtualenv usage, python dependency. Those kinds of things.

File: Formula/pulumi.rb (hunk 1 above)
Comment (stack72) [reply Formula/pulumi.rb:37]: All gone!

**Comment by stack72** [comment]
Hi all

These dependencies have been removed

Paul

**Review by dtrodrigues — APPROVED** [review]
Thanks for your patience and contribution to Homebrew @stack72!

**Pull request closed without merging**